modules/tutorials/pages/tutorial-create-database.adoc

```diff
@@ -0,0 +1,77 @@
+= Create a Cluster
+:description: Create a cluster and use buckets, scopes, and collections to partition your data.
+:page-topic-type: tutorial
+:imagesdir: ../images
+:page-pagination: full
+:page-aliases: buckets-scopes-and-collections.adoc
+
+[abstract]
+{description}
+
+== Buckets, Scopes, and Collections
+
+To organize and manage your data in Couchbase, you can create buckets, scopes, and collections inside your cluster.
+
+A bucket is similar to a cluster, while scopes and collections are used to provide separation between documents of different types.
+A bucket can hold any number of scopes, and a scope can hold any number of collections.
+
+[plantuml,couchbase-hierarchy,svg]
+....
+include::partial$diagrams/couchbase-hierarchy.puml[]
+....
+
+[horizontal]
+bucket:: Stores and retrieves data in the server.              
+scope:: Stores collections. When you create a new bucket, Couchbase provides you with a default scope called `_default`.
+collection:: Contains a set of documents. Couchbase provides you with a default collection called `_default`.
+
+To continue this tutorial, you must create a bucket to hold all student data, a scope to narrow down the data into only data related to an art school, and two collections to narrow it down further into art school students and art school courses.
+
+For more information about buckets, scopes, and collections, see xref:cloud:data-service:about-buckets-scopes-collections.adoc[Buckets, Scopes, and Collections] in the Capella documentation.
+
+
+== Create a Bucket
+
+To create a bucket:
+
+. Go to menu:Dashboard[Buckets].
+. Click btn:[Add Bucket].
+. For the bucket name, enter `student-bucket`.
+. Click btn:[Add Bucket] to create your bucket.
+
+
+== Create a Scope
+
+To create a scope:
+
+. Go to menu:Buckets[Scopes & Collections].
+. Click btn:[Add Scope].
+. For the scope name, enter `art-school-scope`.
+. Click btn:[Save] to create your scope.
+
+
+== Create Collections
+
+For this tutorial, create two collections: one to store student records, and one to store course details.
+
+To create a collection:
+
+. Go to menu:Buckets[Scopes & Collections].
+. Click btn:[Add Collection] next to `art-school-scope`.
```

Comment:
There's no Buckets > Scopes & Collections command. Also, I don't think *Add Collection* is a button (but this is arguable).

```suggestion
. In the list of scopes and collections, next to `art-school-scope`, click *Add Collection*.
```